### ISSUE TITLE
Not all layers in caffe.exe when builded with Cmake on Windows

### ISSUE THREAD

**Issue by Solin1292** [Microsoft/caffe#77]
Epsent layer ImageData. Cmake generated project, MSVS 2013, with linker options to link dependencies of caffe.lib
(no errror When builded with Caffe.sln from Windows folder)

log:
F0426 16:46:44.356539 26460 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: ImageData (known types: AbsVal, Ac
curacy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Em
bed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, InfogainLoss, InnerProduct, Input, LRN, Log, MVN, MemoryData, Multinomi
alLogisticLoss, PReLU, Pooling, Power, Python, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWit
hLoss, Split, TanH, Threshold, Tile)

**Comment by Solin1292** [comment]
resolved by enabling OpenCV via CMake
